sunpy/util/scraper.py

```diff
@@ -262,7 +266,12 @@ def _ftpfileslist(self, timerange):
                         if (datehref >= timerange.start and
                                 datehref <= timerange.end):
                             filesurls.append(fullpath)
-        filesurls = ['ftp://anonymous:data@sunpy.org@' + url[domain + 2:] for url in filesurls]
+
+        if self.appendLogin:
+            filesurls = ['ftp://anonymous:data@sunpy.org@' + url[domain + 2:] for url in filesurls]
+        else:
+            filesurls = ['ftp://' + url[domain + 2:] for url in filesurls]
```

Comment:
Let's make this a bit DRY-er
```suggestion
        login = ""
        if self.appendLogin:
            login = "anonymous:data@sunpy.org@"
        filesurls = [f'ftp://{login}' + url[domain + 2:] for url in filesurls]
```